WordPress/twentytwentyfive
# Add headers

**Description**
Add four header patterns. This PR does not include the headers that are sidebars.
Partial for https://github.com/WordPress/twentytwentyfive/issues/37

**Testing Instructions**
Activate Gutenberg. Note that I am testing with Gutenberg trunk.
If you don't want to test with Gutenberg, then just switch the header patterns manually by inserting them, since you wont see the "Design" panel.

Since the font sizes and font family is not in yet, the main thing to test in this PR is that:
- The blocks are the same as [in the designs.](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3056-195&t=fWImLHb7Ke2yTwcj-0)
- The spacing to the top, bottom, left and right of the header content matches the designs in Figma closely.

Apply the PR.
Confirm that the default header is used and displays correctly in the editor and front.
Test with different screen widths.

Then select the different heading patterns from the Design panel in the sidebar in the Site Editor, and repeat the test.
There is no need to test the patterns that are added by core. The patterns from the theme should be the four first patterns, at the top of the Design panel.
